server/server/server.go

```diff
@@ -403,6 +412,12 @@ func (s *Server) stopLocalModeServer() {
 		_ = s.localModeServer.Close()
 		s.localModeServer = nil
 	}
+
+	//Remove Socket File on Stop
```

Comment:
I don't think the comment is needed, the code is self-explanatory.

Reply:
Removed it